docs/getting-started/tutorial-sample-data.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[tutorial-sample-data]]
+== Exploring {kib} using sample data
+
+You’re new to {kib} and want to try it out, but don’t have any data. No worries. 
+The Flights sample data set can help you get comfortable visualizing {es} data.
+
```

Comment:
Maybe work in the notion of "touring" Kibana with the sample flight data? I think I'd put the emphasis on the one-click set up so you can see how things work, vs not having data/getting "comfortable".